post_install.template

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BREW_HOME="${0%/*}"
+[[ -x $BREW_HOME/bin/brew ]] || \
+  (echo "This script should be located inside Linuxbrew directory."; exit 1)
+
+ORIG_BREW_HOME="_ORIG_BREW_HOME_"
```

Comment:
Add a comment saying that _ORIG_BREW_HOME is being replaced.
Also could use something like `{{ orig_brew_home }}` as the string to replace to make it look like a template argument in popular templating languages.